src/Services/ExperimentationService.cs

```diff
@@ -1,21 +1,35 @@
 ﻿// Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+using System.Xml;
 using DevHome.Common.Contracts;
 using DevHome.Common.Models;
 using DevHome.Common.Services;
+using DevHome.Common.TelemetryEvents;
+using DevHome.Telemetry;
 
 namespace DevHome.Services;
 
 public class ExperimentationService : IExperimentationService
 {
     private readonly ILocalSettingsService _localSettingsService;
+    private readonly Microsoft.Internal.Windows.DevHome.Helpers.Experimentation.Experiment _experimentHelper = new();

```

Comment:
nit: can we include the namespace in the  header `using`, or the type becomes ambiguous?

Reply:
we can, I just thought "experiment" might be ambiguous enough.  But we can always fix it in the future when that happenes.